docs/source/contents/4_draft_mapping.rst

```diff
@@ -1221,6 +1221,85 @@ typeOfResource
 classification
 ==============
 
+Use case
+--------
+Some of our resources have already been formally cataloged and have a classification number. When these are available,
+they are included in the MODS metadata. Serials like the Alumnus and many of the Athletics media guides are good examples.
+Some collections, like the University of Tennessee Commencements collection include full shelfLocators in the classification
+field (e.g. LD5297 .U55 2013). These should be edited before migration.
+
+Justification
+-------------
+This information is helpful to include as it provides information about where the physical item is shelved (though this
+is not a complete shelfLocator) and the broad subject the materials relate to.
+
+Xpath
+-----
+
+mods:classification[@authority="lcc"] OR mods:classification
+
+Decision
+--------
+
+`Example record without authority - tenngirl:977 <https://digital.lib.utk.edu/collections/islandora/object/tenngirl:977/datastream/MODS>`_
+
+.. code-block:: xml
+
+    <classification>LD5296 .W6</classification>
+
+`Example record with authority - agrtfhs:2275 <https://digital.lib.utk.edu/collections/islandora/object/agrtfhs:2275/datastream/MODS>`_
+
+.. code-block:: xml
+
+    <classification authority="lcc">S1 .T43</classification>
+
+.. code-block:: turtle
+
+    @prefix classSchemes: <http://id.loc.gov/vocabulary/classSchemes> .
```

Comment:
The namespace here is wrong:

```
@prefix classSchemes: <http://id.loc.gov/vocabulary/classSchemes/> .

```

Reply:
@markpbaggett - thanks for this and for chatting in Slack for a second about this. I have corrected the issue.